cmd/bnsd/app/app_test.go

```diff
@@ -50,113 +41,56 @@ func TestApp(t *testing.T) {
 
 	// ensure 3 keys with proper values
 	if assert.Equal(t, 3, len(dres.Tags), "%#v", dres.Tags) {
-		// three keys we expect, in order
-		keys := make([][]byte, 3)
-		vals := [][]byte{[]byte("s"), []byte("s"), []byte("s")}
-		hexWllt := []byte("776C6C743A")
-		hexSigs := []byte("736967733A")
-		keys[0] = append(hexSigs, []byte(addr.String())...)
-		keys[1] = append(hexWllt, []byte(addr.String())...)
-		keys[2] = append(hexWllt, []byte(addr2.String())...)
-		if bytes.Compare(addr2, addr) < 0 {
-			keys[1], keys[2] = keys[2], keys[1]
+		wantKeys := []string{
+			toHex("cash:") + addr.String(),
```

Comment:
We are replacing `wllt:` with `cash:`. Does this mean that iov-core has to be updated as well, for example https://github.com/iov-one/iov-core/blob/master/packages/iov-bns/src/tags.ts#L14 ?

Reply:
@husio yes, this breaks current clients. Do the semantics of that field change or is it just a renaming? Please make sure this is listed in a changelog.